### ISSUE TITLE
add github integration to python environment

### ISSUE THREAD

**Issue by janCstoffregen** [nie-ine/inseri#680]
@OmniaKahla fyi

just put the link to github to the file

**Comment by OmniaKahla** [comment]
@janCstoffregen 
Do you mean the python code?

**Comment by janCstoffregen** [comment]
yes, the link to the python code file on Github, then you just push the changes to github and inseri will pull it, so all the bugs regarding that belong to the past then and it's much easier to programm the microservices. Would you mind setting up a Github organisation for Gossembrot, then we can add all the repos there and link them to inseri.
